examples/new-postprocess/main.js

```diff
@@ -0,0 +1,339 @@
+(function() {
+    'use strict';
+
+    var OSG = window.OSG;
+    var osgViewer = OSG.osgViewer;
+    var osgUtil = OSG.osgUtil;
+    var osgShader = OSG.osgShader;
+
+    var osgDB = OSG.osgDB;
+    var requestFile = osgDB.requestFile;
+
+    var osg = OSG.osg;
+
+    var P = window.P;
+
+    var Example = function() {
+        this._composer = undefined;
+        this._shaderProcessor = undefined;
+        this._camera = undefined;
+
+        this._composerConfigNames = ['passthrough.json'];
+        this._composerConfigFiles = {};
+        this._composerConfigFiles['passthrough.json'] = [
+            {
+                func: 'passthrough',
+                textures: ['%last'],
+                out: { name: '%next' }
+            }
+        ];
+
+        var shaders = {};
+        shaders['passthrough.glsl'] =
+            'vec4 passthrough() { return vec4(TEXTURE_2D_TextureInput(gTexCoord).rgb,1.0);}';
+
+        this._shaderProcessor = new osgShader.ShaderProcessor();
+        this._shaderProcessor.addShaders(shaders);
+
+        this._params = {
+            composer: this._composerConfigNames[0]
+        };
+
+        this._lutTextureNames = [
+            'lookup-tables/ue4_neutralLUT.png',
+            'lookup-tables/ue4_lut2.jpg',
+            'lookup-tables/ue4_lut3.jpg',
+            'lookup-tables/2strip.jpg',
+            'lookup-tables/3strip.jpg',
+            'lookup-tables/wtf.jpg'
+        ];
+
+        this._lutTextures = {};
+    };
+
+    Example.prototype = {
+        run: function() {
+            this._viewer = new osgViewer.Viewer(document.getElementById('View'));
+            this._viewer.init();
+            this._viewer.setSceneData(this._createScene());
+            this._viewer.setupManipulator();
+            this._viewer.getManipulator().computeHomePosition();
+            this._viewer.run();
+
+            this._rebuildGUI('passthrough.json');
+        },
+
+        _createCamera: function(width, height, texture) {
```

Comment:
we don't need all the methods to be private. Just for coherence with other examples